pom.xml

```diff
@@ -17,19 +17,33 @@
   </properties>
 
   <dependencies>
+
```

Comment:
Remove unnecessary blank line.

Reply:
Waiting for response on previous PR about this - will change in both places if you would like! Just let me know :)

Reply:
Yes. Please change in both places. New line at the beginning or end of blocks are unnecessary.